app/views/devise/registrations/edit.html.erb

```diff
@@ -1,58 +1,62 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<div class="col-md-6 col-md-push-3">
+  <h2>Edit <%= resource_name.to_s.humanize %></h2>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
 
-  <div class="field">
-    <%= f.label :first_name %><br />
-    <%= f.text_field :first_name, autofocus: true %>
+  <div id="error_explanation" class="col-md-12">
+    <%= devise_error_messages! %>
   </div>
 
-  <div class="field">
-    <%= f.label :last_name %><br />
-    <%= f.text_field :last_name %>
-  </div>
+  <div class="col-md-12">
+    <div class="field">
+      <%= f.label :first_name %><br />
+      <%= f.text_field :first_name, autofocus: true %>
+    </div>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
-  
-  <div class="field">    
-    <%= form_for :profile do |profile_form| %>
-    <%=profile_form.label :introduction %>
-    <p><%= profile_form.text_field :introduction %></p>
-    <%=profile_form.label :birthday %>
-    <p><%= profile_form.date_select :birthday %></p>
-    <% end %>
-  </div>
+    <div class="field">
+      <%= f.label :last_name %><br />
+      <%= f.text_field :last_name %>
+    </div>
 
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
-  <% end %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email %>
+    </div>
 
-  <div class="field">
-    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
-    <%= f.password_field :password, autocomplete: "off" %>
-  </div>
+    <div class="field">
+      <%= form_for :profile do |profile_form| %>
```

Comment:
I think that this also should be changed to `f.fields_for` as in `registrations/new.html.erb`